api/pkg/utils/up9Utils.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+type GuestToken struct {
+	Token string `json:"token"`
+	Model string `json:"model"`
+}
+
+type ModelStatus struct {
+	LastMajorGeneration float64 `json:"lastMajorGeneration"`
+}
+
+func getGuestToken(url string, target *GuestToken) error {
```

Comment:
`getGuestToken` name is a little confusing.
Maybe `getRequestBody`?
Or don't separate it at all from `CreateAnonymousToken`.